### PR TITLE
allow validation override

### PR DESCRIPTION
**Pull request by jh-bate**
also updated field name to be `Verified`

@kentquirk I added a config value that will allow us to override the user verification check for local and developer testing ease. I made it so you *have to* specify the config in order to override so its not there by default.

Let me know what you think

### CODE REVIEW

File: api/shorelineApi.go [hunk 1]
```diff
@@ -23,8 +23,10 @@ type (
 		ServerSecret         string `json:"serverSecret"` //used for services
 		LongTermKey          string `json:"longTermKey"`
 		LongTermDaysDuration int    `json:"longTermDaysDuration"`
-		Salt                 string `json:"salt"`      //used for pw
-		Secret               string `json:"apiSecret"` //used for token
+		Salt                 string `json:"salt"`                //used for pw
+		Secret               string `json:"apiSecret"`           //used for token
+		VerificationSkipped  bool   `json:"verificationSkipped"` //allows for the skipping of verification for testing
```
Comment (kentquirk) [inline api/shorelineApi.go]: I don't think that VerificationSkipped (past tense) is the right name for this. VerificationSkippable, perhaps, or maybe CanSkipVerification would be better.

Or perhaps you could eliminate this variable entirely and just say that if VerificationSecret is either missing or an empty string, you *cannot* skip verification.

Comment (jh-bate) [inline api/shorelineApi.go]: Yeah I agree on the name. Initially I had it with just the secret but there was an issue with that approach I am trying to remember

**Comment by kentquirk** [comment]
Ok, if leaving the canSkip thing is the best thing to do, then please change its name. Other than that, this looks good and you can merge at will.

**Comment by jh-bate** [comment]
@kentquirk I remembered why I had it there and solved the issues so the PR no longer requires that extra config variable

**Pull request merged**